test/e2e/parallel/web_console_plugin_test.go

```diff
@@ -0,0 +1,122 @@
+package parallel
+
+import (
+	"crypto/tls"
+	"fmt"
+	"io"
+	"net/http"
+	"strings"
+	"testing"
+
+	. "github.com/codeready-toolchain/toolchain-e2e/testsupport"
+	v1 "github.com/openshift/api/route/v1"
+	"github.com/stretchr/testify/require"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+func TestWebConsoleDeployedSuccessfully(t *testing.T) {
+	// given
+	await := WaitForDeployments(t)
+
+	for i, memberAwait := range await.AllMembers() {
+
+		signupRequest := NewSignupRequest(await).
+			Username(fmt.Sprintf("consoletest%d", i)).
+			Email("consoletest@redhat.com").
+			TargetCluster(memberAwait).
+			ManuallyApprove().
+			EnsureMUR().
+			RequireConditions(ConditionSet(Default(), ApprovedByAdmin())...).
+			Execute(t)
+		response := signupRequest.GetSignupResponse(t)
+
+		// Response should contain a ConsoleURL with a value something like:
+		// https://console-openshift-console.apps.99b682869228f7464338-mgmt.ci.hypershift.devcluster.openshift.com/
+		require.Contains(t, response, "consoleURL")
```

Comment:
Is the case correct? `ConsoleURL` in the comment vs `consoleURL` in the assertion

Reply:
I have no clue why this check was introduced at all TBH 
https://github.com/codeready-toolchain/toolchain-e2e/pull/690#discussion_r1166832887
I would maybe just drop it entirely as it doesn't seem to have any impact on the rest of the test.

Reply:
anyway, I removed it seems to be unnecessary and just extra noise in the test d25ccfe1
cc @sbryzak